quinn-proto/src/crypto.rs

```diff
@@ -453,7 +454,13 @@ impl PacketNumberKey {
                 let nonce = GenericArray::from_slice(sample);
                 Aes128Ctr::new(key, nonce).apply_keystream(in_out)
             }
-            _ => unimplemented!(),
+            ChaCha20(key) => {
+                let counter = io::Cursor::new(&sample[..4]).get_u32_be();
```

Comment:
`byteorder::BE::read_u32(&sample[..4])`

Reply:
Hmm, yeah. Can't wait for int_to_from_bytes.